package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "cvat",
+  "version": "1.0.0",
+  "description": "Powerful and efficient Computer Vision Annotation Tool (CVAT)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gauharbains/cvat.git"
```

Comment:
Can we change all urls here to point to LabShare/cvat instead of your forked repository?